Duty List Helper/Helper 1/readme.md

```diff
@@ -0,0 +1 @@
+Helper1: To help quickly produce the dates for data.json
```

Comment:
Codacy has a fix for the issue: [Files should end with a single newline character](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8389313)
```suggestion
Helper1: To help quickly produce the dates for data.json

```